src/Customer.php

```diff
@@ -115,6 +115,45 @@ public function lineItems()
         return $line;
     }
 
+    /**
+     * Gets a payment source object for this customer.
+     *
+     * @return PaymentSource
+     */
+    public function payment_sources()
+    {
+        $source = new PaymentSource($this->_client);
+        $source->setEndpointBase($this->getEndpoint());
+
+        return $source;
+    }
+
+    /**
+     * Gets a card object for this customer.
+     *
+     * @return Card
+     */
+    public function cards()
```

Comment:
What are `cards()` and `bank_accounts()` used for?

Reply:
They would be used to build a complete endpoint like  /customers/{customer_id}/{card|bank_account}/{id} so you would be able to delete a payment source by ID without first listing them